Blake Stone - Aliens of Gold.sh.new

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+if [ -d "/opt/system/Tools/PortMaster/" ]; then
+  controlfolder="/opt/system/Tools/PortMaster"
+elif [ -d "/opt/tools/PortMaster/" ]; then
+  controlfolder="/opt/tools/PortMaster"  
+else
+  controlfolder="/roms/ports/PortMaster"
+fi
+
+source $controlfolder/control.txt
+
+# on my rg351mp with amberelec and the roms on the 2nd sd-card
+directory="/storage/roms"
```

Comment:
ok, it actually comes from the `control.txt`. but why did it now work for me? :thinking:

Reply:
ah, found it. in `control.txt`, the value is set to `roms`. but there is a leading slash missing. it must be `/roms` :-)

Reply:
@romadu WDYT? :-)